sites/example-project/src/components/ui/Formatting/CurrencyFormatGrid.svelte

```diff
@@ -0,0 +1,109 @@
+<script>
+  import "./format-grid.css";
+  import { blur } from 'svelte/transition';
+  import { defaultExample, formatExample } from "$lib/modules/formats";
+  export let formats;
+  let selectedCurrency;
+</script>
+
```

Comment:
I was actually thinking about adding an optional display name to the formats object. Not sure if it would have helped here.